apps/nextjs-website/react-components/components/PressRelease/PressRelease.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {
+  Box,
+  Typography,
+  Container,
+  Stack,
+  Link,
+  useTheme,
+} from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+
+import {
+  TextColor,
+  ExtraTextColor,
+  BackgroundColor,
+} from '../common/Common.helpers';
+
+const PressRelease = ({
+  eyelet,
+  title,
+  subtitle,
+  body,
+  ctaText,
+  ctaHref,
```

Comment:
```suggestion
  link,
```
Following previous suggestions.